deployment/helm/balloons/values.yaml

```
@@ -42,6 +42,10 @@ resources:
 
 nri:
   patchRuntimeConfig: false
+  # NRI plugins should be considered as part of the container runtime.
+  # Therefore, adding the system-node-critical priority class to the DaemonSet,
+  # could mitigate potential risk in a running system under load.
+  setPodPriorityClassNodeCritical: false
```

Two suggestions:

1. Don't put the new config field under `nri:` but on the top level. I think this is better in line with the other pod level settings like `toleration` and `nodeSelector`
2. Streamline the field name a bit, drop the "set" from the beginning

```suggestion
podPriorityClassNodeCritical: false
```

---

@marquiz I have updated the helm charts as suggested. I have added the "new" value `podPriorityClassNodeCritical` at the end of the `values.yaml` files to indicate that it is similar to the `toleration` and `nodeSelector` values. Please check again. Thank you very much!